kie-spring-boot/kie-spring-boot-autoconfiguration/jbpm-spring-boot-autoconfiguration/pom.xml

```diff
@@ -59,6 +59,12 @@
     <dependency>
       <groupId>org.jbpm</groupId>
       <artifactId>jbpm-workitems-rest</artifactId>
+      <exclusions>
```

Comment:
This modification breaks lead to [NoClassFoundError/ClassNotFoundError](https://github.com/kiegroup/jbpm-work-items/pull/298#issuecomment-2014741681)

Reply:
Ok, thanks, I will take a look. This was done to fix duplicate classes.

Reply:
@baldimir 
TBH - I think it is better to modify "downstream" (i.e., where the failure actually happen) .
I pointed out this modification to explain what causes the issue

Reply:
i agree with https://github.com/kiegroup/droolsjbpm-integration/pull/3044#discussion_r1540599889 it is always better to declare all dependencies you need and do not rely on pulled transitive ones.